katas/langtonant/daniel/dani_ant.test.js

```diff
@@ -0,0 +1,43 @@
+import { findNewdirection  } from './dani_ant';
+
+test ('This part should give new direction to the ant based on color and current position', ()=>{
+    let newDirection = findNewdirection(0, 'up');
+    expect (newDirection).toBe('right');
+
+    newDirection = findNewdirection(0, 'right');
+    expect (newDirection).toBe('down');
+
+    newDirection = findNewdirection(0, 'down');
+    expect (newDirection).toBe('left');
+
+    newDirection = findNewdirection(0, 'left');
+    expect (newDirection).toBe('up');
+
+    newDirection = findNewdirection(1, 'up');
+    expect (newDirection).toBe('left');
+
+    newDirection = findNewdirection(1, 'left');
+    expect (newDirection).toBe('down');
+
+    newDirection = findNewdirection(1, 'down');
+    expect (newDirection).toBe('right');
+
+    newDirection = findNewdirection(1, 'right');
+    expect (newDirection).toBe('up');
+});
+
+import { antMoveCorrectDir } from './dani_ant';
+
+test ('This should move the ant to the correct direction', ()=>{
+    let newRowMove = antMoveCorrectDir('up');
+    expect (newRowMove).toBe(newRowMove--);
```

Comment:
This works, but I would recommend use hardcoded values for expected arguments, otherwise, they can make the tests tricky or hard to read. For example, it would be better do
```
let initialMove = <value>

expect (newRowMove).toBe(7); // just randoming a value, instead of 7, you should set the expected value

newRowMove = initialMove;
// next test
```